src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -26,6 +26,9 @@ internal sealed class SqlClientDiagnosticListener : ListenerHandler
     public const string SqlDataWriteCommandError = "System.Data.SqlClient.WriteCommandError";
     public const string SqlMicrosoftWriteCommandError = "Microsoft.Data.SqlClient.WriteCommandError";
 
+    private const string ContextInfoParameterName = "@opentelemetry_trace_context";
```

Comment:
```suggestion
    private const string ContextInfoParameterName = "@opentelemetry_traceparent";
```

trace_context has `traceparent` and `tracestate`. We only push `traceparent`.